pkg/kubelet/kuberuntime/labels.go

```diff
@@ -104,6 +105,17 @@ func newContainerLabels(container *v1.Container, pod *v1.Pod) map[string]string
 	labels[types.KubernetesPodUIDLabel] = string(pod.UID)
 	labels[types.KubernetesContainerNameLabel] = container.Name
 
+	for _, c := range pod.Spec.InitContainers {
+		if c.Name == container.Name {
+			labels[types.KubernetesContainerTypeLabel] = types.PodInitContainer
+		}
+	}
+	for _, c := range pod.Spec.Containers {
+		if c.Name == container.Name {
+			labels[types.KubernetesContainerTypeLabel] = types.PodContainer
+		}
+	}
```

Comment:
@kkkkun Do we need two loops here? If container type is either types.PodInitContainer or types.PodContainer we can omit one of the loops, I guess.

Reply:
Thanks & fixed